index.bs

```diff
@@ -1194,6 +1194,13 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
                         :   {{AuthenticatorAttestationResponse/attestationObject}}
                         ::  |attestationObject|
 
+                        :   {{AuthenticatorAttestationResponse/transports}}
+                        ::  A sequence of one or more {{AuthenticatorTransport}} values. The first value is set to the transport used to obtain this registration. Further values are appended to indicate other, distinct, transports that the |authenticator| is believed to support.
+
+                            If a user agent does not wish to divulge this information it MAY choose not to include additional transports and MAY choose the first element to be a generic value rather than the actual transport used. But it MUST still include at least one value and it takes the risk that [=RP=] behaviour may be suboptimital.
```

Comment:
> But it MUST still include at least one value

Why it is a MUST?

Reply:
See also the second part of my comment [here](https://github.com/w3c/webauthn/pull/1050#issuecomment-420714358), but basically I'm defining the transports list to be non-empty and telling browsers that they can fabricate plausible values if they want, but they can't return invalid ones.